packages/react-components/react-avatar/src/components/Avatar/useAvatar.tsx

```diff
@@ -36,7 +36,7 @@ export const useAvatar_unstable = (props: AvatarProps, ref: React.Ref<HTMLElemen
     required: true,
     defaultProps: {
       children: getInitials(name, dir === 'rtl', { firstInitialOnly: size <= 16 }),
-      'aria-hidden': true,
+      id: baseId + '__initials',
```

Comment:
It looks like I accidentally put this ID on the icon slot instead of initials--thanks for fixing it! Could you also remove it from the icon slot (delete line 52), since it shouldn't be there?

Reply:
done!